splinepy/_spline.py

```diff
@@ -217,7 +214,7 @@ def whatami(self, how_dare_you):
         --------
         None
         """
-        logging.warning("Spline - Excuse me, you cannot tell me what I am.")
+        self.logw("Excuse me, you cannot tell me what I am.")
```

Comment:
```suggestion
        self._logw("Excuse me, you cannot tell me what I am.")
```
missing `_`?

Reply:
Yeah, my bad, thanks for pointing that out! 👍